Airports/EGKK/SMR/Regions.txt

```diff
@@ -780,7 +837,7 @@ N051.09.26.027 W000.10.32.581
 N051.09.25.689 W000.10.32.504
 N051.09.25.704 W000.10.32.382
 N051.09.21.019 W000.10.30.687
-;P_Island2
+; - P_Island2
```

Comment:
See previous comments, this is adding a notch into the topmost east side island on P. Whilst this is only one additional line, I feel that it adds very little value to the sector file. (Though as it is only one line I may be more easily convinced).